### PR TITLE
added css style and more description on the comments page, see change…

### PR DESCRIPTION
**Pull request by trinitydonohugh**
…s in link below
https://screenshot.googleplex.com/EQeO7vBbw7Y.png -- includes annotation of design changes

### CODE REVIEW

File: portfolio/src/main/webapp/teatime.html [hunk 1]
```diff
@@ -14,11 +14,52 @@
...
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row mb-3">
+        <div class="col">
+          <ul class="nav nav-tabs">
+            <li class="nav-item">
+              <a class="nav-link active" href="/teatime.html">Comments</a>
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/teatime.html:55]: I think teatime and teatime2 aren't very descriptive names, how about tea time comment and tea time guidelines?

Comment (trinitydonohugh) [reply portfolio/src/main/webapp/teatime.html:55]: ok! I'll change their names now. thank you

Comment (pengxianzhe) [reply portfolio/src/main/webapp/teatime.html:55]: I think you also need to change the filenames in links. Also, you still have your old files added in the pull request, so you want to remove them in next commit.

Keeping the main page teatime.html is fine. Just make sure you have descriptive names for comments tab and community guidelines tab (if you also need another html file for comments tab in the future)

Comment (trinitydonohugh) [reply portfolio/src/main/webapp/teatime.html:55]: ok, I just added that to the new commit -- hopefully it worked.

**Pull request merged**